open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -2164,4 +2238,31 @@ PROCEDURE JSONCLOSE
  NEWLINE
 END [PROCEDURE]
 
+PROCEDURE CUERRORHANDLING
+
+ [CHECK ERROR TEXT LENGTH]
```

Comment:
Depending upon what the error being displayed is, could cutting it off at character 40 - no matter what - make it challenging to read?  Maybe consider starting at character 40 and backtracking until a space is found and then cutting the sentence at that point.